src/components/Navigation.astro

```diff
@@ -190,22 +219,26 @@ const featuredSponsors = DATA.sponsors.filter(
 
 <style>
   @media not all and (max-width: 501px) {
-      .drawer-open-transition {
-        transition-duration: 500ms;
-        transform: translate(0, 0);
-      }
+    .drawer-open-transition {
```

Comment:
what do you think of not negating mqs? just like this 

```css
  @media (min-width: 501px) {
    .drawer-open-transition {
      transition-duration: 500ms;
      transform: translate(0, 0);
    }
  }

  @media (max-width: 500px) {
    .drawer-open-transition {
      opacity: 1;
      transform: translate(0, 0);
      transition: opacity 0.3s ease-in-out, transform 0ms ease-in-out;
    }

    .drawer-close-transition {
      opacity: 0;
      transition: opacity 0.25s ease-in-out, transform 0ms ease-in-out 0.25s;
    }
  }
```

Reply:
Yes, much better, I just got confused with some tailwind media-queries, changed at [latest commit](https://github.com/biznaga-fest/web/pull/10/commits/1c79b8e222120cddb4fead217f91b3b50f774d67)